app.py

```diff
@@ -1,31 +1,307 @@
-import pymongo
 import os
-from flask import Flask
+from flask import Flask, render_template, redirect, request, url_for, jsonify, flash
+from flask_pymongo import PyMongo, pymongo
+from bson.objectid import ObjectId
+from bson import SON
+from pprint import pprint
+from pymongo import ASCENDING, DESCENDING, TEXT
+from functions import get_pages, generate_pagination_links, get_countries
+from flask_wtf import FlaskForm, Form
+from wtforms import  TextField, SelectField, TextAreaField, validators, StringField, SubmitField
 
 app = Flask(__name__)
+app.secret_key = 'some_secret'
 
-@app.route('/')
-def hello():
-    return 'Hello World'
+app.config["MONGO_DBNAME"] = 'cookbookdatabase'
+app.config["MONGO_URI"] ="mongodb://admin:Komplete8@ds119150.mlab.com:19150/cookbookdatabase"
+
+mongo = PyMongo(app)
+
+country_list = get_countries()
+
+class ReusableForm(Form):
```

Comment:
Create a file called `forms.py` and create all your forms there. then import it here

Reply:
This has been updated.